.appveyor.yml

```diff
@@ -9,9 +9,9 @@ install:
   - ps: Install-Product node $env:nodejs_version
   - yarn install
   - if "%nodejs_version%" == "7" (
-      yarn run lint &&
-      yarn run coverage  &&
-      yarn run test-doclint
+      yarn lint &&

```

Comment:
let's do the opposite and convert all `yarn` into `npm`:
- everybody who knows what `yarn` is also knows what `npm` is
- the opposite is not true

Reply:
Hm, I do not think this cleanup-focused PR would be the proper place to address changing the package manager for the project as well as deciding which is better suited for the project.

Such a change/discussion would be better suited to a separate `chore` PR/Issue in my opinion. Granted, this specific appveyor stanza is skirting the line between chore and docs.

Either way, that change would entail a massive PR replacement of `yarn.lock` with a `package-lock.json` that would need to be verified to be equivalent to prevent any potential build errors, which seems way out-of-scope for this current PR.

Reply:
> Such a change/discussion would be better suited to a separate chore PR/Issue in my opinion.

@KennyRules agreed. This is addressed in #1878.

Reply:
Awesome, thanks!